main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western",)
+    Genre.objects.create(name="Action",)
+    Genre.objects.create(name="Dramma",)
```

Comment:
The genre name 'Dramma' seems to be a typo. It is later corrected to 'Drama' in line 18, but it's better to insert the correct name initially.